.pre-commit-config.yaml

```diff
@@ -20,3 +20,13 @@ repos:
     hooks:
       - id: black
         language_version: python3.11
+
+  - repo: local
```

Comment:
i tried using the mypy pre-commit mirror but was having issues with the type stubs we need (like types-orjson, types-PyYaml, etc) so just running it directly